src/components/layout/PageHeader.tsx

```
@@ -0,0 +1,44 @@
+'use client'
+
+// import { Search } from 'lucide-react'
+import { ThemeToggle } from '@/components/theme/ThemeToggle'
+import { useRequireAuth } from '@/hooks/useAuth'
+
+interface PageHeaderProps {
+  onRefresh?: () => void
+  showSearch?: boolean
+}
+
+export function PageHeader({}: PageHeaderProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)